client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useLayoutEffect } from 'react';
+import { Button, CircularProgress, IconButton, Typography, Skeleton, Grid } from '@mui/material';
+import { Box } from '@mui/system';
+import DeleteForeverIcon from '@mui/icons-material/DeleteForever';
+import { Formik, FormikHelpers } from 'formik';
+import AvatarDisplay from '../../../components/AvatarDisplay/AvatarDisplay';
+import SettingHeader from '../../../components/SettingsHeader/SettingsHeader';
+import { User } from '../../../interface/User';
+
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import { makeStyles } from '@mui/styles';
+import editProfile from '../../../helpers/APICalls/editProfile';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
```

Comment:
Most of these styles can be applied to the `Box` component directly.

Eg: `<Box display="flex" alignItems="center"...>`

Ditto where applicable!